docs/linting/typed-linting/MONOREPOS.md

```diff
@@ -62,6 +62,32 @@ module.exports = {
 };
 ```
 
+### Wide globs in `parserOptions.project`
+
+Using wide globs `**` in your `parserOptions.project` may cause performance implications. Instead of globs that use `**` to recursively check all folders, prefer paths that use a single `*` at a time.
```

Comment:
Just nitpicking: it's good to be specific (_what_ implications?).

```suggestion
Using wide globs `**` in your `parserOptions.project` may degrade linting performance.
Instead of globs that use `**` to recursively check all folders, prefer paths that use a single `*` at a time.
```